pkg/console/operator/sync_v400.go

```diff
@@ -341,9 +347,9 @@ func (co *consoleOperator) SyncConfigMap(
 		}
 	}
 
-	pluginsEndpoingMap := co.GetPluginsEndpointMap(operatorConfig.Spec.Plugins)
-
-	defaultConfigmap, _, err := configmapsub.DefaultConfigMap(operatorConfig, consoleConfig, managedConfig, infrastructureConfig, activeConsoleRoute, useDefaultCAFile, inactivityTimeoutSeconds, pluginsEndpoingMap)
+	pluginsEndpointMap := co.GetPluginsEndpointMap(operatorConfig.Spec.Plugins)
+	managedClusters := co.GetMangagedClusters(ctx)
+	defaultConfigmap, _, err := configmapsub.DefaultConfigMap(operatorConfig, consoleConfig, managedConfig, infrastructureConfig, activeConsoleRoute, useDefaultCAFile, inactivityTimeoutSeconds, pluginsEndpointMap, managedClusters)
```

Comment:
Not related to this change, but we will need to review and refactor this approach since adding parameters is not a great long term solution.